openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -35,9 +35,15 @@ class ArnoldStandinLoader(load.LoaderPlugin):
     color = "orange"
 
     def load(self, context, name, namespace, options):
+        if not cmds.pluginInfo("mtoa", query=True, loaded=True):
+            cmds.loadPlugin("mtoa")
+            # create defaultArnoldRenderOptions before creating aiStandin
+            # which tried to connect it. Since we load the plugin and directly
+            # create aiStandin without the defaultArnoldRenderOptions,
+            # here needs to create the render options for aiStandin creation.
```

Comment:
```suggestion
            # Create defaultArnoldRenderOptions before creating aiStandin
            # which tries to connect it. Since we load the plugin and directly
            # create aiStandin without the defaultArnoldRenderOptions,
            # we need to create the render options for aiStandin creation.
```